### ISSUE TITLE
Release fishtree 0.3.1

### ISSUE THREAD

**Issue by jonchang** [jonchang/fishtree#5]
Prepare for release: 
 
* [x] `devtools::check()` 
* [x] `devtools::check_win_devel()` 
* [x] `rhub::check_for_cran()` 
* [x] `revdepcheck::revdep_check(num_workers = 4)` 
* [x] [Polish NEWS](https://style.tidyverse.org/news.html#news-release) 
 
Submit to CRAN: 
 
* [x] `usethis::use_version('patch')` 
* [x] Update `cran-comments.md` 
* [x] `devtools::submit_cran()` 
* [x] Approve email 
 
Wait for CRAN... 
 
* [x] Accepted :tada: 
* [x] `usethis::use_github_release()` 
* [x] `usethis::use_dev_version()` 
* [ ] Tweet

**Comment by jonchang** [comment]
* no reverse dependencies

**Comment by jonchang** [comment]
* checking CRAN incoming feasibility ... NOTE
Maintainer: 'Jonathan Chang <jonathan.chang@ucla.edu>'

Version contains large components (0.3.0.9000)

* checking examples ... NOTE
Examples with CPU or elapsed time > 5s
                   user system elapsed
fishtree_alignment 0.88   0.03   13.56

**Comment by jonchang** [comment]
https://builder.r-hub.io/status/fishtree_0.3.0.9000.tar.gz-7521441946c84b6fabfe26a9dd42013d
   https://builder.r-hub.io/status/fishtree_0.3.0.9000.tar.gz-0573969af76b4a909b6206cdad7ab68b
   https://builder.r-hub.io/status/fishtree_0.3.0.9000.tar.gz-9f5fab78aef649f7ab1b9358fcc891f4

**Comment by jonchang** [comment]
win-builder:

```
* checking Rd line widths ... NOTE
Rd file 'fishtree_phylogeny.Rd':
  \examples lines wider than 100 characters:
     genomic_fish <- c("Oryzias latipes", "Tetraodon nigroviridis", "Gasterosteus aculeatus", "Danio rerio")
```

fixed in e30d8c7580a708214f59974e5ae59432b57fc0cf
